src/components/AutocompleteField/autocomplete-field.scss

```diff
@@ -0,0 +1,48 @@
+@import 'globals';
+
+.root {
+  position: relative;
+  margin-bottom: map-get($spacing, 'm-bottom');
+}
+
+.root:last-child {
+  margin-bottom: 0;
+}
+
+.wrapper {
+  position: relative;
+}
+
+.icon {
+  position: absolute;
+
+  @include media('<=800px') {
+    bottom: 100%;
+    margin-bottom: 10px;
+    right: 0;
+  }
+
+  @include media('>800px') {
+    left: 100%;
+    margin-left: 15px;
+    top: 0;
+  }
+}
+
+.message {
+  line-height: 20px;
+  min-height: 20px;
+  margin-top: 8px;
+}
+
+.valid {
+  border-color: color('grey');
+}
+
+.valid:focus {
+  border-color: color('dimmed');
+}
+
+.invalid {
+  border-color: color('error');
+}
```

Comment:
A lot of repetition from the TextField component, but I don‘t think I have the choice?